lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingTransportServer.cs

```diff
@@ -35,8 +33,7 @@ namespace Org.Apache.REEF.Wake.Remote.Impl
     /// Server to handle incoming remote messages.
     /// </summary>
     /// <typeparam name="T">Generic Type of message. It is constrained to have implemented IWritable and IType interface</typeparam>
-    [Obsolete("Need to remove Iwritable and use IstreamingCodec. Please see Jira REEF-295 ", false)]
-    public class WritableTransportServer<T> : IDisposable where T : IWritable
+    public class StreamingTransportServer<T> : IDisposable
```

Comment:
The usual question: This seems like an implementation detail. Does it have to be `public`?